tonic/src/transport/server/tls.rs

```diff
@@ -56,11 +58,24 @@ impl ServerTlsConfig {
         }
     }
 
+    /// Sets whether the server's cipher preferences are followed instead of the client's.
+    /// It prevents attacks such as POODLE
```

Comment:
I don't think that tonic should maintain the document about some specific use cases same as rustls does not do.

Reply:
@tottoto , ok sure. Shall I just remove line 62?